docs/semgrep-language-reference/semgrep-rule-schema.yaml

```
@@ -0,0 +1,836 @@
+        type: string
+      as:
+        type: string
+      renames:
+        type: array
+        items:
+          $ref: "#/$defs/joinRename"
+
+  joinObject:
+    type: object
+    additionalProperties: false
+    required: [on]
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)